### ISSUE TITLE
GG.GG is no longer accessible

### ISSUE THREAD

**Issue by MattTheTekie** [738/awesome-url-shortener#61]
I'm not sure if this still works, but I'm unable to connect to the gg.gg site.

**Comment by mtxadmin** [comment]
Yes, DNS says "Server failed". But not "Non-existent domain" :-)

**Comment by 738** [comment]
Thanks for the report/suggestion. Resolved in merged PR #162.

Moved gg.gg out of the active shortener list because its homepage now shows a Good Game placeholder.

[Source](https://gg.gg) · [Full review](https://github.com/738/awesome-url-shortener/blob/7de08712716f498797563e4cabaaf8230879b087/BACKLOG_REVIEW.md).
